test/image/mocks/gl3d_surface_after_heatmap.json

```diff
@@ -2,28 +2,59 @@
     "data": [
       {
         "type": "heatmap",
-        "x": [0, 1, 2],
-        "y": [0, 1, 2],
+        "x": [0, 1, 2, 3, 4],
```

Comment:
Why did you patch this mock?

Reply:
This was a new mock added quite recently. I thought we could expand a bit to allow interactivelytesting `hovertitle` on gl3d & gl2d.